hw/ip/keymgr/data/keymgr.prj.hjson

```diff
@@ -13,6 +13,12 @@
         life_stage:         "L1",
         design_stage:       "D1",
         verification_stage: "V1",
```

Comment:
If you increment the version number, can you add a commit hash for the previous version (or not increment the version number).

Reply:
thanks @imphil i'll do that.